pyo3-derive-backend/src/pymethod.rs

```diff
@@ -631,8 +650,7 @@ pub fn impl_py_method_def_static(spec: &FnSpec, wrapper: &TokenStream) -> TokenS
     }
 }
 
-pub fn impl_py_class_attribute(spec: &FnSpec<'_>, wrapper: &TokenStream) -> TokenStream {
-    let python_name = &spec.python_name;
```

Comment:
I understand why you did this refactoring but I think I preferred how it was before, to keep symmetry with the the other `impl_py_method_xyz` functions.

Reply:
I hesitated making this change indeed. I’ll write two functions, it’ll duplicate code slightly but will preserve the symmetry.